content/docs/studio/user-guide/projects-and-experiments/visualize-and-compare.md

```diff
@@ -31,37 +30,48 @@ Iterative Studio can work with two types of plots files in your repository:
 2. Image files in JPEG, GIF, PNG, or SVG format. These images will be displayed
    directly in Iterative Studio.
 
-You can define multiple plots in a single repository. Below is an example
-snippet from a `dvc.yaml` file showing the `evaluate` stage of the DVC pipeline.
+You can define multiple plots in a single repository. The plots can be
+[stage-specific](/doc/user-guide/experiment-management/visualizing-plots#plot-outputs)
+or
+[top-level](/doc/user-guide/experiment-management/visualizing-plots#top-level-plots).
+Below is an example snippet from a `dvc.yaml` file showing the `evaluate` stage
+of the DVC pipeline, with 2 different plots. It also includes a top-level plot
+which uses data from the file `runtime_logs/logs.csv`.
```

Comment:
```suggestion
You can define plots in one or more `dvc.yaml` files. These can either be
[pipeline outputs](/doc/user-guide/experiment-management/visualizing-plots#plot-outputs)
or
[top-level](/doc/user-guide/experiment-management/visualizing-plots#top-level-plots).
definitions. Below is a sample `dvc.yaml` file with a plot in the
`evaluate` stage and aanother at the top-level:
```

Reply:
@jorgeorpinel I've incorporated the changes, but with some modification. PTAL.

Reply:
Thanks